tilelang/language/eager/builder.py

```
@@ -725,7 +728,10 @@ def source(self) -> str:
         return self.ir_gen.source
 
     def __call__(self, *args: _P.args, **kwargs: _P.kwargs) -> _T:
-        builder = Builder.current() or Builder()
+        builder = Builder.current()
+        if builder is None:
+            raise JITNoBuilderError("T.macro can only be used inside @tilelang.jit")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)